<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayWaySetPage</class>
 <widget class="QWidget" name="PayWaySetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="skinWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#skinWidget{
	background-color: #ffffff;
}</string>
     </property>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>238</y>
        <width>124</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分的元整方式：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPageButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>15</width>
        <height>18</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>18</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>18</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-image: url(:/kPosSetImage/right.png);
}

QPushButton:pressed{
border-image: url(:/kPosSetImage/right_press.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>124</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>是否启用：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="GridWidget" name="firstGrideWidget" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>400</width>
        <height>81</height>
       </rect>
      </property>
      <widget class="SwitchButton" name="memberStartButton">
       <property name="geometry">
        <rect>
         <x>92</x>
         <y>10</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="SwitchButton" name="alipayStartButton">
       <property name="geometry">
        <rect>
         <x>172</x>
         <y>10</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="SwitchButton" name="weChatMemberButton">
       <property name="geometry">
        <rect>
         <x>252</x>
         <y>50</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="SwitchButton" name="weChatStartButton">
       <property name="geometry">
        <rect>
         <x>252</x>
         <y>10</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="SwitchButton" name="alipayMemberButton">
       <property name="geometry">
        <rect>
         <x>172</x>
         <y>50</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>79</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>是</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>79</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>是</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>79</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>否</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="SwitchButton" name="virtualMemberButton">
       <property name="geometry">
        <rect>
         <x>332</x>
         <y>50</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="SwitchButton" name="virtualButton">
       <property name="geometry">
        <rect>
         <x>332</x>
         <y>10</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>124</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>是否用于会员支付：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>400</width>
        <height>71</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="payImageLabel_0">
        <property name="geometry">
         <rect>
          <x>28</x>
          <y>7</y>
          <width>29</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/kPosSetImage/money.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="payMethodLabel_0">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>42</y>
          <width>79</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>现金</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="payMethodLabel_1">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>42</y>
          <width>79</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>会员卡</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="payImageLabel_1">
        <property name="geometry">
         <rect>
          <x>104</x>
          <y>7</y>
          <width>29</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/kPosSetImage/huiyuanka.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="payMethodLabel_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>42</y>
          <width>79</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>支付宝</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="payImageLabel_2">
        <property name="geometry">
         <rect>
          <x>187</x>
          <y>7</y>
          <width>29</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/kPosSetImage/zhifubao.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="payImageLabel_3">
        <property name="geometry">
         <rect>
          <x>265</x>
          <y>7</y>
          <width>29</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/kPosSetImage/wxzhifu.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="payMethodLabel_3">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>42</y>
          <width>79</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>微信支付</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="payMethodLabel_4">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>42</y>
          <width>79</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>虚拟支付</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="payImageLabel_4">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>7</y>
          <width>29</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/kPosSetImage/xuni.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>41</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>京东钱包</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_35">
        <property name="geometry">
         <rect>
          <x>32</x>
          <y>1</y>
          <width>35</width>
          <height>35</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/kPosSetImage/jd_wallet.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_34">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>41</y>
          <width>32</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>pay</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>1</y>
          <width>32</width>
          <height>39</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/kPosSetImage/apple_pay.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="KPosComboBox" name="cashComBox" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>236</y>
        <width>81</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QPushButton</extends>
   <header>SwitchButton.h</header>
  </customwidget>
  <customwidget>
   <class>KPosComboBox</class>
   <extends>QWidget</extends>
   <header>KPosComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GridWidget</class>
   <extends>QWidget</extends>
   <header location="global">GridWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
